src/test/java/seedu/realodex/logic/commands/CommandTestUtil.java

```diff
@@ -28,8 +28,10 @@
  */
 public class CommandTestUtil {
 
-    public static final String VALID_NAME_AMY = "Amy Bee";
-    public static final String VALID_NAME_BOB = "Bob Choo";
+    public static final String VALID_NAME_AMY_CAPS = "Amy Bee";
+
+    public static final String VALID_NAME_AMY_NON_CAPS = "amy bee";
+    public static final String VALID_NAME_BOB_CAPS = "Bob Choo";
```

Comment:
Suggestion: Would it be ok to include tests also for all caps/irregular cased names? "AMY CHOO"/"aMY chOO" should both be parsed to "Amy Choo". To do this, perhaps you can consider changing the capitalizeWord method to toLower() everything first before capitalising the first word 😄

Reply:
Good catch, my caps function was rather wrong too

Reply:
Changed function and added new commits